main.go

```diff
@@ -1,7 +1,6 @@
 package main
 
 import (
-	"code.google.com/p/gcfg"
```

Comment:
Those were removed in https://github.com/maurizi/otm-ecoservice/commit/e69f34f1ab6b408823ff3f98ef890a731c2945f4